### PR TITLE
Add new version of dependency analytics extension v0.1.0 to plugin registry as a latest version

### PR DESCRIPTION
**Pull request by dvandra**
### What does this PR do?
Adding new version of dependency-analytics extension v0.1.0 as a latest version.

### CODE REVIEW

**Comment by dvandra** [comment]
@arajkumar PTAL

File: v3/plugins/redhat/dependency-analytics/0.1.0/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+displayName: Dependency Analytics
+title: Insights about your application dependencies
+description: Insights about your application dependencies, Security, License compatibility and AI based guidance to choose appropriate dependencies for your application.
+icon: https://www.eclipse.org/che/images/logo-eclipseche.svg
+repository: https://github.com/fabric8-analytics/fabric8-analytics-vscode-extension
+category: Other
+firstPublicationDate: '2020-07-14'
+spec:
+  containers:
+    - image: "quay.io/eclipse/che-sidecar-dependency-analytics:0.0.13-a38cb0c"
```
Comment (ericwill) [inline v3/plugins/redhat/dependency-analytics/0.1.0/meta.yaml:15]: Are you okay with using this container sidecar? The new version of the plugin doesn't need any new dependencies?

Comment (dvandra) [reply v3/plugins/redhat/dependency-analytics/0.1.0/meta.yaml:15]: Yeah, it is working fine with the existing container sidecar. The new major updates are mostly for vulnerability data improvement by integration with Snyk and others are on back-end API side.

**Comment by dvandra** [comment]
Thanks @ericwill

**Comment by arajkumar** [comment]
@ericwill Is there a chance to cherry-pick this on 7.16.x release branch?

**Comment by ericwill** [comment]
No I believe it's too late as 7.16.2 is already out. It will be in 7.17 which is out next Wednesday.

**Comment by arajkumar** [comment]
Thanks @ericwill . That should be okay for us.

**Pull request merged**